core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -71,64 +72,122 @@ object ConsumerGroupCommand {
       }
     } catch {
       case e: Throwable =>
-        println("Error while executing consumer group command " + e.getMessage)
-        println(Utils.stackTrace(e))
+        consumerGroupOutputWriter.printError("Executing consumer group command failed due to " + e.getMessage, Some(e))
     } finally {
       consumerGroupService.close()
     }
   }
 
+  case class PartitionAssignmentState(group: String, topic: Option[String], partition: Option[Int], offset: Option[Long],
+                                      lag: Option[Long], memberId: Option[String], memberHost: Option[String],
+                                      clientId: Option[String], logEndOffset: Option[Long])
+
+  trait OutputWriter {
+    def printError(msg: String, e: Option[Throwable] = None)
+    def printConfirmation(msg: String)
+    def printAssignment(groupAssignment: Array[PartitionAssignmentState])
+  }
+
+  class ConsumerGroupOutputWriter extends OutputWriter with Logging {
+
+    def printError(msg: String): Unit = println("Error: " + msg)
+
+    def printError(msg: String, e: Option[Throwable] = None): Unit = {
+      println("Error: " + msg)
+      if (e.isDefined)
+        debug(Utils.stackTrace(e.get))
```

Comment:
Minor: if we use debug(), do we need to extract the stack trace? Maybe you could do something like this:
```scala
e.map(debug("Exception in consumer group command", _))
```